circle.yml

```diff
@@ -39,11 +39,8 @@ machine:
 
 checkout:
   post:
-    - |
-      git clone --depth 1 ${ST2_PACKAGES_REPO} /home/ubuntu/st2/st2-packages
-      cd /home/ubuntu/st2/st2-packages
-      git checkout ${CIRCLE_BRANCH} || true
-    - .circle/buildenv.sh
+    - git clone -b ${CIRCLE_BRANCH} --single-branch --depth 1 ${ST2_PACKAGES_REPO} /home/ubuntu/st2/st2-packages
+    - /home/ubuntu/st2/st2-packages/.circle/buildenv.sh
```

Comment:
Also FYI every command in Circle is run in separate subshell, so any vars or `cd` are not preserved across different command, see: https://circleci.com/docs/configuration#phases